content/en/docs/tasks/administer-cluster/memory-manager.md

```diff
@@ -0,0 +1,390 @@
+
+In other words, the example above illustrates that for the conventional memory (`type=memory`), we reserve `3Gi` in total, i.e.: 
+
+`sum(reserved-memory(i)) = reserved-memory(0) + reserved-memory(1) = 1Gi + 2Gi = 3Gi`
+
+An example of Node Allocatable Feature flags [configuration](https://kubernetes.io/docs/tasks/administer-cluster/out-of-resource/):
+* `--kube-reserved=cpu=500m,memory=50Mi`
+* `--system-reserved=cpu=123m,memory=333Mi`
+* `--eviction-hard=memory.available<500Mi`
+
+_NOTICE: hard eviction threshold is not equal to zero by default but `100Mi`, so do not forget to decrease the total amount set via `--reserved-memory` by this `100Mi`. Otherwise, the Memory Manager will display an error. Here is an example of a correct configuration:_
+
+```
+--feature-gates=MemoryManager=true 
+--kube-reserved=cpu=4,memory=4Gi 
+--system-reserved=cpu=1,memory=1Gi 
+--memory-manager-policy=static 
+--reserved-memory 0:memory=3Gi --reserved-memory 1:memory=2148Mi
+```
+Let us validate the configuration above:
+1. `kube-reserved + system-reserved + eviction-hard(default) = reserved-memory(0) + reserved-memory(1)`
+2. `4Gi + 1Gi + 100Mi = 3Gi + 2148Mi`
+3. `5120Mi + 100Mi = 3072Mi + 2148Mi`
+4. `5220Mi = 5220Mi` (correct!)
+
+## Placing a Pod in the Guaranteed QoS class
+
+If the selected policy is anything other than `none`, Memory Manager would consider Pods in Guaranteed QoS class and provide topology hints to the Topology Manager. Otherwise, the Memory Manager returns default topology hint to the Topology Manager.
+
+The following pod specifications lend a Pod to Guaranteed QoS class.
+
+```yaml
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    resources:
+      limits:
+        memory: "200Mi"
+        cpu: "2"
+        example.com/device: "1"
+      requests:
+        memory: "200Mi"
+        cpu: "2"
+        example.com/device: "1"
+```
+
+This pod with integer CPU request runs in the `Guaranteed` QoS class because `requests` are equal to `limits`.
+
+```yaml
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    resources:
+      limits:
+        memory: "200Mi"
+        cpu: "300m"
+        example.com/device: "1"
+      requests:
+        memory: "200Mi"
+        cpu: "300m"
+        example.com/device: "1"
+```
+
+Also, this pod with sharing CPU request runs in the `Guaranteed` QoS class because `requests` are equal to `limits`.
+
+Notice that both CPU and memory requests must be specified for a Pod to lend it to Guaranteed QoS class.
+
+## Troubleshooting
+
+The following means can be used to troubleshoot the reason why a pod could not be deployed or became rejected at a node:
+- pod status - indicates topology affinity errors
+- system logs - include valuable information for debugging, e.g., about generated hints
+- state file - the dump of internal state of the Memory Manager (includes [Node Map and Memory Maps][2]) 
+
+### Pod status (TopologyAffinityError)

```

Comment:
### Pod status (TopologyAffinityError) {#TopologyAffinityError}

Reply:
Is this syntax defined somewhere? Could you provide a link?
e.g. `"{#TopologyAffinityError}"` or `"{{< note >}}"`

Reply:
This makes the fragment identifier become `TopologyAffinityError`, which I think makes sense.

The syntax for setting a custom fragment identifier is mentioned in https://gohugo.io/content-management/cross-references/#heading-ids